examples/notedump/notedump_main.c

```diff
@@ -0,0 +1,263 @@
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ *****************************************************************************/
+
+/*****************************************************************************
+ * Included Files
+ *****************************************************************************/
+
+#include "notedump.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <arpa/inet.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+
+#include <unistd.h>
+#include <string.h>
+#include <errno.h>
+
+#include <netinet/in.h>
+#include <fcntl.h>
+
+#define NOTE_MSG_SIZE CONFIG_SYSTEM_NOTE_BUFFERSIZE
+
+/*****************************************************************************
+ * Public Data
+ *****************************************************************************/
+
+/*****************************************************************************
+ * Private Data
+ *****************************************************************************/
+
+static uint32_t g_udpserver_ipv4;
+static uint32_t g_udpserver_port;
+static uint32_t note_sent_bytes;
+
+/*****************************************************************************
+ * Private Functions
+ *****************************************************************************/
+
+/*****************************************************************************
+ * show_usage
+ *****************************************************************************/
+
+static void show_usage(FAR const char *progname)
+{
+  fprintf(stderr, "USAGE: %s [<server-addr>] [<server-port>]\n", progname);
+  exit(1);
+}
+
+/*****************************************************************************
+ * udp_cmdline
+ *****************************************************************************/
+
+bool udp_cmdline(int argc, char **argv)
+{
+  int ret;
+  long port;
+  char *endptr;
+
+  /* Init the default IP address */
+
+  g_udpserver_ipv4 = HTONL(0x01020304);
+  g_udpserver_port = HTONS(1234);
+
+  /* Currently only a single command line option is supported:  The server
+   * IP address. Used to override default.
+   */
+
+  if (argc == 3)
+    {
+      /* Convert the <server-addr> argument into a binary address */
+
+      ret = inet_pton(AF_INET, argv[1], &g_udpserver_ipv4);
+      if (ret == 1)
+        {
+          fprintf(stderr, "Server address: %s\n", argv[1]);
+        }
+      else
+        {
+          fprintf(stderr, "ERROR: Invalid server address: %s\n", argv[1]);
+          show_usage(argv[0]);
+          return false;
+        }
+
+        port = strtol(argv[2], &endptr, 10);
```

Comment:
```suggestion
      port = strtol(argv[2], &endptr, 10);
```
Fixing the indentation.